homeassistant/components/frontend/templates/index.html

```diff
@@ -9,8 +9,8 @@
     <link rel='apple-touch-icon' sizes='180x180'
           href='/static/icons/favicon-apple-180x180.png'>
     <link rel="mask-icon" href="/static/icons/home-assistant-icon.svg" color="#3fbbf4">
-    <link rel='preload' href='{{ core_url }}' as='script'/>
     {% if not dev_mode %}
+      <link rel='preload' href='{{ core_url }}' as='script'/>
       {% for panel in panels.values() -%}
         <link rel='prefetch' href='{{ panel.webcomponent_url }}'>
```

Comment:
We should do more research if this actually really matters. I think that we should also add the webfonts here, as those are actually critical.